### DISCUSSION TITLE
Why is there no "option" button after enabling

### DISCUSSION THREAD

**Discussion by Fanarrr** [ivan-lednev/obsidian-day-planner#477] [Q&A]
<img width="1379" alt="pic" src="https://github.com/ivan-lednev/obsidian-day-planner/assets/85116689/971da86c-58b0-46ff-afc5-f0e33a5dabe4">

**Comment by Fanarrr** [comment]
Tried re-installing many times. Dayplanner-og version has an “option” button and other plugins are working properly as well.  
Can't figure out why. 🥲

**Comment by ivan-lednev** [comment]
Honestly, I have no idea 🤷‍♂️  It shows up on my Obsidian. You can get to the settings screen from the side panel:
![image](https://github.com/ivan-lednev/obsidian-day-planner/assets/41428836/d0ef04ea-9209-4e17-8e49-4ec8a5d5fb3c)
